src/Cli/GenerateInvoicesCommand.php

```diff
@@ -31,7 +31,11 @@ public function __construct(
     public function execute(InputInterface $input, OutputInterface $output): void
     {
         /** @var array $orders */
-        $orders = $this->orderRepository->findAll();
+        $orders = $this->orderRepository
```

Comment:
Shouldn't we also skip orders in state `cart`? /cc @Zales0123

Reply:
I thought order in cart will not have order number yet.

Reply:
That's correct :D nvm then